opba-embedded-starter/src/main/resources/application.yml

```diff
@@ -74,7 +74,7 @@ api:
   # Common cookie template
   cookie:
     secure: false # localhost-only - TODO application-dev.yml for dev env?
-    max-age: PT300S
+    max-age: PT5M # PLEASE SYNC api.security.token-validity-duration, api.cookie.max-age and facade.expirable.expire-after-write
```

Comment:
Keep comment but you can reference the value directly:
```
 max-age: ${api.security.token-validity-duration}
```

Reply:
Done. 
Whats about facade.expirable.expire-after-write?